pkg/log/context/context.go

```diff
@@ -0,0 +1,23 @@
+package context
```

Comment:
Instead of this move the global `Log` inside the logger package and reference `Log = logger.DefaultLog`. This should avoid the cycling dep and keep packages simple no?

Reply:
addressed